ui/imports/shared/controls/chat/UserImage.qml

```diff
@@ -24,7 +24,11 @@ Loader {
 
     signal clicked()
 
-    height: active && item ? item.height : 0
+    height: {
+        // I dont know why, the binding doesn't work if this isn't here
+        item && item.height
+        return this.active ? item.height : 0
+    }
```

Comment:
Could you please try to remove this line completely and see if it works?

I was there before https://github.com/status-im/status-desktop/pull/6050/commits/a436ae2f86aec0068ac8dfd429c1e27b7fbe85be but it seems it wasn't merged to master :sweat: 

The problem here is the "binding race" I believe:
```
Loader {
    active: someCondition
    height: active ? item.height : 0
    sourceComponent: Rectangle { height: 100 }
}
```
what will be the height of Rectangle when someCondition is set to true? well.. it depends if the component is loaded faster than the height binding is resolved :dizzy_face: 

> every usage of height: active ? item.height : 0 is bad for Loaders, it will lead to nasty bugs as you presented me today

that's the message I've sent to Sale 06/10/2022 :rofl:

Reply:
Yeah, it should not be needed (ideally); the Qt docu says:

`
If an explicit size is not specified for the Loader, the Loader is automatically resized to the size of the loaded item once the component is loaded.
`

Reply:
You're right, it wasn't needed.